gemfiles/ruby_3.2.0_contrib_old.gemfile

```diff
@@ -37,5 +37,6 @@ gem "google-protobuf", ["~> 3.0", "!= 3.7.0", "!= 3.7.1"]
 gem "sorbet", "= 0.5.9672"
 gem "spoom", "~> 1.1"
 gem "dalli", "< 3.0.0"
+gem "graphql", ">= 1.12.0", "< 2.0"
```

Comment:
Just wondering if this was expected in this changeset.

Reply:
Same as https://github.com/DataDog/dd-trace-rb/pull/1993#discussion_r861551459 😄